### PR TITLE
Implement Configure Public Client

### LINKED ISSUE #169: Configurations client package
**Is your feature request related to a problem? Please describe.**
Implement Configurations client package

**Describe the solution you'd like**
Implement Configurations client package so Configurations endpoints are available

**Describe alternatives you've considered**
NA

**Additional context**
API already implemented

### PR DESCRIPTION
**Pull request by xoanmi**
Basic implementation for the `configure` client.

Close #169

### CODE REVIEW

**Comment by xoanmi** [comment]
@elenz97 could you please check this PR?

**Review by elenz97 — CHANGES_REQUESTED** [review]

File: apiv2/pkg/clients/configure/configure.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	return &RESTClient{
+		Options:  opts,
+		V2Client: v2Client,
+		AuthInfo: authInfo,
+	}
+}
+
+type Client interface {
+	GetConfigs(ctx context.Context) (*model.ConfigurationsResponse, error)
+	GetInternalConfigs(ctx context.Context) (*model.InternalConfigurationsResponse, error)
```
[the remark this answers is not in the document]
Comment (xoanmi) [reply apiv2/pkg/clients/configure/configure.go]: Removed as it is not used anyway

File: apiv2/pkg/clients/configure/configure_integration_test.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+	require.Equal(t, "db_auth", *resp.AuthMode)
+}
+
+func TestAPIUpdateConfigs(t *testing.T) {
+	authMode := "oidc"
+	ctx := context.Background()
+	c := NewClient(clienttesting.V2SwaggerClient, clienttesting.DefaultOpts, clienttesting.AuthInfo)
+
+	err := c.UpdateConfigs(ctx, &modelv2.Configurations{AuthMode: authMode})
```
[the remark this answers is not in the document]
Comment (xoanmi) [reply apiv2/pkg/clients/configure/configure_integration_test.go:30]: I will remove this integration test! The GetInternalConfig API is not intended to be used from users

[the remark this answers is not in the document]
Comment (xoanmi) [reply apiv2/pkg/clients/configure/configure_integration_test.go:30]: Done!

[the remark this answers is not in the document]
Comment (elenz97) [reply apiv2/pkg/clients/configure/configure_integration_test.go:30]: `authMode` still seems not be a pointer in this test :see_no_evil:

**Review by elenz97 — CHANGES_REQUESTED** [review]

File: apiv2/pkg/clients/configure/configure_integration_test.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (xoanmi) [reply apiv2/pkg/clients/configure/configure_integration_test.go]: Sorry, I missed it. Now it should be fixed. I used the same auth_method in both tests to keep consistency

**Review by elenz97 — APPROVED** [review]
Looks good! Thanks for your contribution, @xoanmi

**Pull request merged**